css/style.css

```diff
@@ -0,0 +1,68 @@
+   body{
+        background-color: lightblue;
+        text-align: center;
+        
+      }
+
```

Comment:
- [ ] It would be good to use body tags the design style such as "height, width, and position"  properties are used to set the height and width of an element, and position properties dictate how the browser should arrange elements on the page.